File: December - 09/day9.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int n;
    cin >> n;
    vector<string> input(n);

    for (auto &x : input)
        cin >> x;
    int max_len = INT_MIN;

    string word;
    for (auto e : input)
    {
        int len = e.size();
        if (e.size() % 2 != 0)
        {
            max_len = max(len, max_len);
        }
        else
            continue;
    }
    for (auto e : input)
    {
        if (e.size() == max_len)
        {
            word = e;
            break;
        }
    }
    if (max_len == INT_MIN)
        cout << "Better Luck Next Time";
    else
        cout << word;

    return 0;
}

File: December - 09/day 9.py

import math
n=int(input())
list_A=list(input().split())
len_arr=[]
ind_arr=[]
for i in range(0,n):
   if(len(list_A[i])%2==1):
       len_arr.append(len(list_A[i]))
       ind_arr.append(i)

if(len_arr==[]):
    print("Better luck next time")
else:
    max=-math.inf
    for i in range(0,len(len_arr)):
        if len_arr[i]>max:
            max=len_arr[i]
            word=ind_arr[i]
    print(list_A[word])    